quarto/intro.qmd

```diff
@@ -8,7 +8,76 @@
 
 ### Render vs preview (Guilherme / Estelle)
 
-## Computations (Mahesh / Guilherme)
+## Computations
+
+Quarto lets you perform computations within your notebook. This is typically
+done using code blocks denoted by three backticks followed by the language 
+you're using in curly brackets.
+
+````{.markdown}
+```{{python}}
+1 + 1
+```
+````
+
+Quarto supports computations in several languages:
+
+- [Python](https://quarto.org/docs/computations/python.html)
+- [R](https://quarto.org/docs/computations/r.html)
+- [Julia](https://quarto.org/docs/computations/julia.html)
+- [Observable](https://quarto.org/docs/computations/ojs.html)
+
+Additional languages can also be supported through other Jupyter kernels (see Engines below).
+See this page for a list of [Jupyter kernels](https://gist.github.com/chronitis/682c4e0d9f663e85e3d87e97cd7d1624).
+
+The languages and packages used in your computations must be available in your `render` environment, and 
+are often installed through other means, for example using `conda` or a container platform.
+
+Packages and variables created within one code block are also accessible from other code blocks, including
+inline code blocks. For example, here we create a figure and reference the value in the caption.
+
+````{.markdown}
```

Comment:
Even though the section is not about references per se, could be nice to include the markdown code to actually reference a figure? E.g., "see @fig-plot-alt for a plot of $y=x^2$.

Reply:
Updated text and made it more about the referencing.